test/AdaptivityTests/GridTransferTests.jl

```diff
@@ -0,0 +1,131 @@
+module GridTransferTests
```

Comment:
I would also add grid transfer tests for the multifield FE case. 

If am not wrong, we are testing all this machinery only for the SingleFieldFESpace case, right?

In principle, all should work verbatim by composability, but better to test it so that there are no surprises.

Reply:
I've added some tests, and everything works. I'll try to add some more concerning integration, but it should also work first try.